<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1385</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1381</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 237, 228);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter un  produit</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-40</y>
        <width>1391</width>
        <height>781</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(232, 232, 232);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="reference_produit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font:  75 16pt &quot;Times New Roman&quot;;
color: rgb(44, 51, 89);
</string>
       </property>
       <property name="text">
        <string>referenc_ produit </string>
       </property>
      </widget>
      <widget class="QLabel" name="prix">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font:  75 16pt &quot;Times New Roman&quot;;
color: rgb(44, 51, 89);</string>
       </property>
       <property name="text">
        <string>prix</string>
       </property>
      </widget>
      <widget class="QLabel" name="date_produit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font:  75 16pt &quot;Times New Roman&quot;;
color: rgb(44, 51, 89);</string>
       </property>
       <property name="text">
        <string>Date_produit</string>
       </property>
      </widget>
      <widget class="QLabel" name="quantite">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font:  75 16pt &quot;Times New Roman&quot;;
color: rgb(44, 51, 89);</string>
       </property>
       <property name="text">
        <string>Quantite</string>
       </property>
      </widget>
      <widget class="QLabel" name="nom">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font:  75 16pt &quot;Times New Roman&quot;;
color: rgb(44, 51, 89);</string>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_reference_produit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Styles du QLineEdit avec des couleurs différentes */
QLineEdit {
    background-color: #dfe6e9; /* Gris clair */
    box-shadow: 2px 0.5px 5px 1px inset #000000;
    border-radius: 10px;
    width: 70%;
    padding-right: 10px;
    padding-left: 10px;
    text-align: left;
    color: #2C3E50; /* Bleu foncé */
    font-size: inherit;
    font-family: Arial, Helvetica, sans-serif;
    font-weight: normal;
    font-variant: normal;
    border-color: #bdc3c7; /* Gris légèrement plus foncé */
}

QLineEdit::hover {
    color: #000000; /* Noir */
    background-color: #ecf0f1; /* Gris clair au survol */
}

QLineEdit::focus {
    color: #000000; /* Noir */
    background-color: #ffffff; /* Blanc */
}
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prix">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>120</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Styles du QLineEdit avec des couleurs différentes */
QLineEdit {
    background-color: #dfe6e9; /* Gris clair */
    box-shadow: 2px 0.5px 5px 1px inset #000000;
    border-radius: 10px;
    width: 70%;
    padding-right: 10px;
    padding-left: 10px;
    text-align: left;
    color: #2C3E50; /* Bleu foncé */
    font-size: inherit;
    font-family: Arial, Helvetica, sans-serif;
    font-weight: normal;
    font-variant: normal;
    border-color: #bdc3c7; /* Gris légèrement plus foncé */
}

QLineEdit::hover {
    color: #000000; /* Noir */
    background-color: #ecf0f1; /* Gris clair au survol */
}

QLineEdit::focus {
    color: #000000; /* Noir */
    background-color: #ffffff; /* Blanc */
}
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_date_produit">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>160</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Styles du QLineEdit avec des couleurs différentes */
QLineEdit {
    background-color: #dfe6e9; /* Gris clair */
    box-shadow: 2px 0.5px 5px 1px inset #000000;
    border-radius: 10px;
    width: 70%;
    padding-right: 10px;
    padding-left: 10px;
    text-align: left;
    color: #2C3E50; /* Bleu foncé */
    font-size: inherit;
    font-family: Arial, Helvetica, sans-serif;
    font-weight: normal;
    font-variant: normal;
    border-color: #bdc3c7; /* Gris légèrement plus foncé */
}

QLineEdit::hover {
    color: #000000; /* Noir */
    background-color: #ecf0f1; /* Gris clair au survol */
}

QLineEdit::focus {
    color: #000000; /* Noir */
    background-color: #ffffff; /* Blanc */
}
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>260</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Styles du QLineEdit avec des couleurs différentes */
QLineEdit {
    background-color: #dfe6e9; /* Gris clair */
    box-shadow: 2px 0.5px 5px 1px inset #000000;
    border-radius: 10px;
    width: 70%;
    padding-right: 10px;
    padding-left: 10px;
    text-align: left;
    color: #2C3E50; /* Bleu foncé */
    font-size: inherit;
    font-family: Arial, Helvetica, sans-serif;
    font-weight: normal;
    font-variant: normal;
    border-color: #bdc3c7; /* Gris légèrement plus foncé */
}

QLineEdit::hover {
    color: #000000; /* Noir */
    background-color: #ecf0f1; /* Gris clair au survol */
}

QLineEdit::focus {
    color: #000000; /* Noir */
    background-color: #ffffff; /* Blanc */
}
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_Ajouter">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>410</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Styles de base du QPushButton */
QPushButton {
    background-color: #CCCCCC; /* Gris clair */
    color: #000000; /* Noir */
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #FF0000; /* Rouge */
    border-radius: 8px;
    padding: 10px;
    transition: background-color 0.3s ease; /* Transition en douceur de la couleur de fond */
}

/* Animation de survol */
QPushButton:hover {
    background-color: #AAAAAA; /* Gris un peu plus foncé au survol */
}
</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Downloads/1091585.png</normaloff>../../../Downloads/1091585.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer_3">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>410</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Styles de base du QPushButton */
QPushButton {
    background-color: #CCCCCC; /* Gris clair */
    color: #000000; /* Noir */
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #FF0000; /* Rouge */
    border-radius: 8px;
    padding: 10px;
    transition: background-color 0.3s ease; /* Transition en douceur de la couleur de fond */
}

/* Animation de survol */
QPushButton:hover {
    background-color: #AAAAAA; /* Gris un peu plus foncé au survol */
}
</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Downloads/6861362.png</normaloff>../../../Downloads/6861362.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>410</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Styles de base du QPushButton */
QPushButton {
    background-color: #CCCCCC; /* Gris clair */
    color: #000000; /* Noir */
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #FF0000; /* Rouge */
    border-radius: 8px;
    padding: 10px;
    transition: background-color 0.3s ease; /* Transition en douceur de la couleur de fond */
}

/* Animation de survol */
QPushButton:hover {
    background-color: #AAAAAA; /* Gris un peu plus foncé au survol */
}
</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Downloads/6325975.png</normaloff>../../../Downloads/6325975.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_1">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>500</y>
         <width>321</width>
         <height>161</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../Downloads/Sans_titre-2-removebg-preview.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTableView" name="tab_produits">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>140</y>
         <width>611</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    border:none;
    font: 75 28pt &quot;Arial&quot;;
}

QTableView {
    background-color: #f5f5f5;
    alternate-background-color: #e0e0e0;
    selection-background-color: #a8d8ea;
    selection-color: #333;
    gridline-color: #ccc;
}

QHeaderView::section {
    background-color: #FF0000; /* Rouge */
    color: white;
    padding: 5px;
    border: 1px solid #ddd;
}

QTableView::item {
    padding: 5px;
}

QTableView::item:selected {
    background-color: #a8d8ea;
}

QTableView::item:hover {
    background-color: #d4e9f8;
}

QScrollBar:vertical {
    border: 1px solid #ddd;
    background: #FF0000; /* Rouge */
    width: 10px;
}

QScrollBar::handle:vertical {
    background: #FF0000; /* Rouge */
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    border: 1px solid #ddd;
    height: 10px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar:horizontal {
    border: 1px solid #ddd;
    background: #FF0000; /* Rouge */
    height: 10px;
}

QScrollBar::handle:horizontal {
    background: #FF0000; /* Rouge */
}

QScrollBar::add-line:horizontal,
QScrollBar::sub-line:horizontal {
    border: 1px solid #ddd;
    width: 10px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}
</string>
       </property>
      </widget>
      <widget class="QLabel" name="nom_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
font:  75 16pt &quot;Times New Roman&quot;;
color: rgb(44, 51, 89);</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_email">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>320</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Styles du QLineEdit avec des couleurs différentes */
QLineEdit {
    background-color: #dfe6e9; /* Gris clair */
    box-shadow: 2px 0.5px 5px 1px inset #000000;
    border-radius: 10px;
    width: 70%;
    padding-right: 10px;
    padding-left: 10px;
    text-align: left;
    color: #2C3E50; /* Bleu foncé */
    font-size: inherit;
    font-family: Arial, Helvetica, sans-serif;
    font-weight: normal;
    font-variant: normal;
    border-color: #bdc3c7; /* Gris légèrement plus foncé */
}

QLineEdit::hover {
    color: #000000; /* Noir */
    background-color: #ecf0f1; /* Gris clair au survol */
}

QLineEdit::focus {
    color: #000000; /* Noir */
    background-color: #ffffff; /* Blanc */
}
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_quantite">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>210</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Styles du QLineEdit avec des couleurs différentes */
QLineEdit {
    background-color: #dfe6e9; /* Gris clair */
    box-shadow: 2px 0.5px 5px 1px inset #000000;
    border-radius: 10px;
    width: 70%;
    padding-right: 10px;
    padding-left: 10px;
    text-align: left;
    color: #2C3E50; /* Bleu foncé */
    font-size: inherit;
    font-family: Arial, Helvetica, sans-serif;
    font-weight: normal;
    font-variant: normal;
    border-color: #bdc3c7; /* Gris légèrement plus foncé */
}

QLineEdit::hover {
    color: #000000; /* Noir */
    background-color: #ecf0f1; /* Gris clair au survol */
}

QLineEdit::focus {
    color: #000000; /* Noir */
    background-color: #ffffff; /* Blanc */
}
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="le_pdf">
       <property name="geometry">
        <rect>
         <x>1230</x>
         <y>580</y>
         <width>91</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    font: 8pt &quot;Sitka Display Semibold&quot;;
    text-align: middle;
}

QPushButton:hover {
    color: rgb(255, 255, 255);

    border: none;
    text-align: middle;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Downloads/337946.png</normaloff>../../../Downloads/337946.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="le_tri_2">
       <property name="geometry">
        <rect>
         <x>1180</x>
         <y>290</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Style simplifié du QPushButton */
QPushButton {
    background: #eee; /* Gris clair */
    color: #333; /* Noir */
    border: 2px solid #f00; /* Rouge */
    border-radius: 8px;
    padding: 10px;
    transition: background 0.3s ease; /* Transition en douceur de la couleur de fond */
}

/* Animation de survol */
QPushButton:hover {
    background: #ddd; /* Gris plus foncé au survol */
}
</string>
       </property>
       <property name="text">
        <string> PRIX</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Downloads/tri.png</normaloff>../../../Downloads/tri.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="le_tri">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>350</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Style simplifié du QPushButton */
QPushButton {
    background: #eee; /* Gris clair */
    color: #333; /* Noir */
    border: 2px solid #f00; /* Rouge */
    border-radius: 8px;
    padding: 10px;
    transition: background 0.3s ease; /* Transition en douceur de la couleur de fond */
}

/* Animation de survol */
QPushButton:hover {
    background: #ddd; /* Gris plus foncé au survol */
}
</string>
       </property>
       <property name="text">
        <string>reference_produit</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Downloads/tri.png</normaloff>../../../Downloads/tri.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="stat_butt">
       <property name="geometry">
        <rect>
         <x>1110</x>
         <y>90</y>
         <width>281</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    font: 8pt &quot;Sitka Display Semibold&quot;;
    text-align: middle;
}

QPushButton:hover {
    color: rgb(255, 255, 255);

    border: none;
    text-align: middle;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Downloads/Capture_d_écran_2023-12-01_204721-removebg-preview.png</normaloff>../../../Downloads/Capture_d_écran_2023-12-01_204721-removebg-preview.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnEnregistrer">
       <property name="geometry">
        <rect>
         <x>1310</x>
         <y>610</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    font: 8pt &quot;Sitka Display Semibold&quot;;
    text-align: middle;
}

QPushButton:hover {
    color: rgb(255, 255, 255);
 

    border: none;
    text-align: middle;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Downloads/2810399.png</normaloff>../../../Downloads/2810399.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_recherche">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>80</y>
         <width>501</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">/* Styles de base pour QLineEdit */
QLineEdit {
    background-color: #eee; /* Gris clair */
    color: #000000; /* Noir */
    font-size: 16px;
    font-weight: normal; /* Le style normal est plus approprié pour un QLineEdit */
    border: 2px solid #FF0000; /* Rouge */
    border-radius: 10px;
    padding: 10px;
}

/* Animation de survol */
QLineEdit:hover {
    background-color: #eee /* Gris un peu plus foncé au survol */
}
</string>
       </property>
       <property name="maxLength">
        <number>32775</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>1030</x>
         <y>90</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../Downloads/editfind_104370.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1385</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
